content/en/blog/2024-07-16-helm2to3-becomes-unsupported.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Helm 2to3 is Now Unsupported"
+slug: "helm2to3-becomes-unsupported"
+authorname: "Martin Hickey"
+author: "@hickeyma"
+authorlink: "https://helm.sh"
+date: "2024-07-16"
+---
+
+Over four years ago, we [introduced Helm 3](https://helm.sh/blog/helm-3-released/), a major evolution in Helm's development. And we [announced](https://helm.sh/blog/2019-10-22-helm-2150-released/) at that time that Helm 2 would receive patches and security updates for a year. We also provided a [migration path to Helm 3 from Helm 2](https://helm.sh/docs/topics/v2_v3_migration/) and a tool [helm-2to3](https://github.com/helm/helm-2to3) to automate migration.
```

Comment:
```suggestion
Over four years ago, we [introduced Helm 3](https://helm.sh/blog/helm-3-released/), a major evolution in Helm's development. And, we [announced](https://helm.sh/blog/2019-10-22-helm-2150-released/) at that time that Helm 2 would receive patches and security updates for a year. We also provided a [migration path to Helm 3 from Helm 2](https://helm.sh/docs/topics/v2_v3_migration/) and a tool [helm-2to3](https://github.com/helm/helm-2to3) to automate migration.
```

Reply:
Leave as is, as not sure comma is necessary